src/lib/dom_helpers.js

```diff
@@ -32,7 +43,7 @@ function bindFocusables( instance, activateOnFocus ) {
             if ( onFocusPrev ) onFocusPrev.call( element, event );
           };
         };
-        for ( let element of [ ...focusables ] ) {
+        for ( let element of nodeListToArray(focusables) ) {
           element.onfocus = onFocus( element );
         }
```

Comment:
Apparently it doesn't work in IE < 9, so I went with the above approach
instead; if you're not intending to support IE8 and under then it's
probably fine.

See notes at
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

On Fri, 22 Jul 2016, 23:15 Jed Verity <notifications@github.com> wrote:

> In src/lib/dom_helpers.js
> <https://github.com/glortho/react-keydown/pull/27#discussion_r71876297>:
>
> > @@ -32,7 +43,7 @@ function bindFocusables( instance, activateOnFocus ) {
> >              if ( onFocusPrev ) onFocusPrev.call( element, event );
> >            };
> >          };
> > -        for ( let element of [ ...focusables ] ) {
> > +        for ( let element of nodeListToArray(focusables) ) {
> >            element.onfocus = onFocus( element );
> >          }
>
> Thank you for this. What do you think about taking this approach instead?
>
> Array.prototype.slice.call( focusables ).forEach( element => ...
>
> This should be simpler and produce less code on build.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/glortho/react-keydown/pull/27/files/ef1260bc300cf3d152f1761e3e28df7ea43ffe94#r71876297>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAY4_hRt6rvK_yRAZcqZERblzGYtf5g3ks5qYMKPgaJpZM4JSTdv>
> .
>